flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/Pojo.java

```diff
@@ -18,26 +18,31 @@
 package org.apache.flink.streaming.connectors.cassandra;
 
 import com.datastax.driver.mapping.annotations.Column;
-import com.datastax.driver.mapping.annotations.Table;
 
 import java.io.Serializable;
 
-/** Test Pojo with DataStax annotations used. */
-@Table(keyspace = "flink", name = "test")
+/**
+ * Test Pojo with DataStax annotations created dynamically. This class does not use primitive types
+ * and use public fields because it is manipulated via reflection through a subclass created by
+ * ByteBuddy.
+ */
 public class Pojo implements Serializable {
 
     private static final long serialVersionUID = 1038054554690916991L;
 
     @Column(name = "id")
-    private String id;
+    public String id;
 
     @Column(name = "counter")
-    private int counter;
+    public Integer counter;
```

Comment:
Why don't you use Integer.TYPE for the constructor like I did in the prototype?

Reply:
I also don't think they need to be public.

Reply:
I did not see you were using `Integer.TYPE` in the prototype. Fixed.